api/features/serializers.py

```diff
@@ -593,6 +593,10 @@ class Meta:
         fields = ("id", "feature", "environment")
 
 
+class AssociatedFeaturesQuerySerializer(serializers.Serializer):
+    environment = serializers.IntegerField(required=False)
```

Comment:
Just a note to self — we do seem to have a convention to use `entity`, not `entity_id`, for query filters. For environments, it's only broken for the `evaluation-data` endpoint.